app/views/customer_mailer/bike_possibly_found_email.text.haml

```diff
@@ -1,8 +1,10 @@
 = t(".text.we_think_your_stolen_bike", bike_title: @bike.title_string)
```

Comment:
Could you add the name of the external registry (in bold or italic) to this sentence?